### PR TITLE
NAS-107662 / 21.08 / Remote syslog server is overloaded alert

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/alert.py [hunk 1]
```diff
@@ -328,17 +328,17 @@ async def list_sources(self):
...
+        Text('key', null=True),
         Datetime('datetime'),
         Datetime('last_occurrence'),
         Bool('dismissed'),
         Any('mail', null=True),
         Str('text'),
         Str('id'),
         Str('level'),
-        Str('formatted', null=True),
+        Text('formatted', null=True),
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/alert.py:341]: @sonicaj I suppose we now have middleware failing to return strings with length > 1024.

**Review by yocalebo — CHANGES_REQUESTED** [review]
1. This doesn't take into account the remote syslog connection on HA systems.
2. This request originated in September of 2020 by @anodos325. I'd like to know why this request came in the first place and if this is still even a valid request.

**Comment by themylogin** [comment]
@yocalebo you're right, I forgot about HA syslog.

I'll wait with fixing this until the idea itself gets reviewed.

**Comment by yocalebo** [comment]
Spoke with @anodos325 offline and this can be closed as the request is no longer valid.

**Pull request closed without merging**